front50-s3/src/main/java/com/netflix/spinnaker/front50/model/S3Support.java

```diff
@@ -81,7 +81,11 @@ void startRefresh() {
   }
 
   public Collection<T> all() {
-    if (readLastModified() > lastRefreshedTime || allItemsCache.get() == null) {
+    return all(false);
+  }
+
+  public Collection<T> all(boolean alwaysReadFromCache) {
```

Comment:
`refreshCache` might be clearer to grok than `alwaysReadFromCache` (and reverse the default value).